attrs/utils.go

```diff
@@ -0,0 +1,20 @@
+package attrs
+
+import "strings"
+
+// DataAttr returns the name for a data attribute
+func DataAttr(name string) string {
+	var builder strings.Builder
+	builder.WriteString(DataPrefix)
+	builder.WriteString(name)
+	return builder.String()
+}
+
+// DataAttrs returns Props of data attributes build from given name-value pairs
+func DataAttrs(pairs map[string]string) Props {
```

Comment:
<img width="472" alt="Screenshot 2023-11-17 at 5 54 54 AM" src="https://github.com/chasefleming/elem-go/assets/1666730/03025157-2ab3-4032-8604-72d7e74f8342">

We might need to only have the `DataAttr` and not `DataAttrs` function. Without some sort of props merge I'm not sure this will work and may be confusing to the developer. Let's just simplify with the single `DataAttr` method for now. Sorry I didn't catch this earlier. Thanks.